dubbo-config/dubbo-config-api/src/main/java/org/apache/dubbo/config/bootstrap/DubboBootstrap.java

```diff
@@ -1273,7 +1273,7 @@ private void waitExportFinish() {
         try {
             logger.info(NAME + " waiting services exporting asynchronously...");
             CompletableFuture<?> future = CompletableFuture.allOf(asyncExportingFutures.toArray(new CompletableFuture[0]));
-            future.get();
+            future.get(3, TimeUnit.SECONDS);
```

Comment:
3 seconds might not enough for some exporters

Reply:
I think the future here should equal countdownlatch.await (). The longest export interface should also be completed within this time range.